vcal/postproc/postproc_IFS.py

```diff
@@ -41,10 +41,12 @@
 from vip_hci.fits import open_fits, write_fits
 try:
     from vip_hci.psfsub import pca, pca_annular
+    from vip_hci.psfsub.utils_pca import _compute_stim_map, _compute_inverse_stim_map
```

Comment:
for all occurrences: better to import from metrics. `compute_stim_map` has been renamed `stim_map`,  and `compute_inverse_stim_map` has become `inverse_stim_map`, for consistency with the call to `snr_map`.